docs(sca): documentation related to the new SCA mode (#2455)
bonitasoft/bonita-doc: modules/build-run/pages/run-application.adoc

3. Increment the version.

[CAUTION]
The project version must respect the https://semver.org[Semantic Versioning (SemVer)] specification.

=== Build the project

Build the new version of your Application for a given environment. Please refer to the dedicated documentation on xref:build-run:build-application.adoc[how to build a project].

=== Run the Application

As mentioned above, follow the same instructions on <<run, how to run an Application>>.

[NOTE]
Following SemVer specification, if the new version is lower or equals than the current version, the Application will not be updated.
Only process parameters stored in the Bonita Configuration File will be updated.

=== Update strategies

During the update of an application, the following artifacts are updated according to the following strategies:

[cols="1,2,3,3"]
|===
|Sequence order |Artifact |Update strategy |Comment

|#1
|BDM
|Update the BDM, limited by what is supported (add column & add objects). The Bonita Platform is paused during the process. If the BDM is the same as the one already deployed, the update is ignored as well as the pause.

> |For the Enterprise Edition, BDM Access Controls are uninstall, if present, before updating the BDM.

Reviewer: ```suggestion
|For Subscription editions, BDM Access Controls are uninstall, if present, before updating the BDM.
```

|#2
|Organization
|Merge the organization and deactivate elements that are only present in the previous organization.
|

|#3
|Profiles
|In case of conflict on the name, the profile is not imported and the existing profile is kept unmodified. Otherwise, all others are imported.

> |Enterprise Edition only

Reviewer: ```suggestion
|Subscription editions only
```

|#4
|REST API Extensions
|Update the content of existing REST API Extensions and create the new ones.
|

|#5
|Pages
|Update the content of existing Pages and create the new ones.
|

|#6
|Layouts
|Update the content of existing Layouts and create the new ones.
|

|#7
|Themes
|Update the content of existing Themes and create the new ones.
|

|#8
|Living Applications
|Uninstall the previous versions and install the new ones.
|

|#9
|Processes
|Deactivate the previous versions and activate the new ones (keep open cases with previous versions until they are finished).
|If you want to keep the latest version of a process activated, you should duplicate the process from your project in the Bonita Studio.

|#10
|Bonita Configuration File
|Update existing process parameters and create the new ones. Unknown parameters are ignored.
|A Bonita Configuration File is always imported even after a reboot of the Bonita Platform (i.e. the "new" version of the Application is the same as the current version already installed).

|#11
|BDM Access Controls
|Uninstall the previous versions and install the new ones.

> |Enterprise Edition only

Reviewer: ```suggestion
|Subscription editions only
```
